tests/foreman/cli/test_remoteexecution.py

```diff
@@ -252,6 +252,10 @@ def test_positive_run_default_job_template(self):
             'inputs': 'command="ls"',
             'search-query': "name ~ {0}".format(self.client.hostname),
         })
+        JobInvocation.get_output({
```

Comment:
Same here as in some another PR. Why do we get output and do not make any evaluation of it?